methods/iamx.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "iamx",
+  "status": "registered",
+  "specification": "https://github.com/IAMXID/did-method-iamx/blob/main/IAMX_DID_method.md",
+  "contactName": "IAMX AG, 6300 Zug, Switzerland",
+  "contactEmail": "contact@iamx.id",
```

Comment:
We've recently been trying to establish who the change controllers are for registration entries. Should we assume that anyone who's able to respond to an email with that same domain is allowed to update this entry?